docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -152,7 +156,29 @@ To manage the media controller features in your application, you must learn to h
    mcServerInfo.sendPlaybackState('PAUSE', successCallback, errorCallback);
    ```
 
-## Receiving Notifications from the Server
+## Setting and Getting Server Icon URI
+
+You can update the icon URI address of a Media Controller Server, which can then be accessed
```

Comment:
Is there a specific reason to write "media controller server" as "Media Controller Server"? Throughout the document, we are using "media controller server".  Please verify.

Reply:
not really, but the media-controller guide is not consistent on the use of capital/small letters in media controller. There are instances of both forms.

Reply:
I changed Media Controller Server to media controller server as per @Reni-90 request. Please review and let me know if there is anything else to fix.